solutions/standard/DA-types.md

```diff
@@ -0,0 +1,51 @@
+# Configuring complex inputs for KMS in IBM Cloud projects
+
+Several optional input variables in the IBM Cloud [KMS all inclusive deployable architecture](https://cloud.ibm.com/catalog#deployable_architecture) use complex object types. You specify these inputs when you configure deployable architecture.
+
+* Rules For Context-Based Restrictions (`cbr_rules`)
+
+
+## Rules For Context-Based Restrictions <a name="cbr_rules"></a>
+
+The `cbr_rules` input variable allows you to provide a list of rules for the target service in this case the KMS all inclusive DA instance to enforce access restrictions for the instance based on the context of access requests. Context are criterias that include the network location of access requests, the endpoint type from where the request is sent etc ..
+
+- Variable name: `cbr_rules`.
+- Type: A list of objects. Each object represents a set of rules
```

Comment:
Though this is a list but the list will only have one rule for the target service and not the set of rules.
Issue created for this - https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/issues/364

Reply:
fixed https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/commit/806305d97382a08396e3ba9111e5efdbda2cf2cc